config/templates/hive-csv-template.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
     support: Hive Team
     alm-examples: |-
       [{"apiVersion":"hive.openshift.io/v1","kind":"HiveConfig","metadata":{"name":"hive"},"spec":{"managedDomains":[{"aws":{"credentialsSecretRef":{"name":"my-route53-creds"}},"domains":["my-base-domain.example.com"]}]}}]
+    operators.operatorframework.io/internal-objects: '["checkpoints.hive.openshift.io","clusterdeprovisions.hive.openshift.io","clusterprovisions.hive.openshift.io","clusterstates.hive.openshift.io","machinepoolnameleases.hive.openshift.io","clustersyncleases.hiveinternal.openshift.io","clustersyncs.hiveinternal.openshift.io","fakeclusterinstalls.hiveinternal.openshift.io"]'
```

Comment:
If `"checkpoints.hive.openshift.io","clusterdeprovisions.hive.openshift.io","clusterprovisions.hive.openshift.io","clusterstates.hive.openshift.io","machinepoolnameleases.hive.openshift.io"` are to be considered internal, is there a reason we don't move them to the `hiveinternal` group?

Reply:
Some of them I'd definitely like to be move to internal but the process to do so is astonishingly hard and just not worth it. Others I would consider fine to programatically use like deprovision/provision, clusterstate. leases for sure i'd like in internal and checkpoints maybe.